src/authentication/testAuthProvider.tsx

```diff
@@ -36,7 +40,7 @@ export default class TestAuthProvider implements AuthProvider {
   }
 
   public verifyLogIn(): Promise<void> {
-    return this.refresh();
+    return Promise.resolve();
```

Comment:
Is there a reason for this change? This could cause unexpected behaviour in other tests.

Reply:
Oops, I was meant (but forgot) to revert this change. I can't remember why I actually made the change in the first place. Good spot, thanks!